### ISSUE TITLE
Text labels on plots off center when deploying to shinyapps.io

### ISSUE THREAD

**Issue by Dr-Fishopolis** [davidgohel/ggiraph#168]
I'm using ggiraph for a shiny app that I'm deploying to shinyapps.io. It had been working great until I published the dashboard online, but now I'm running into an unfixable issue where text labels are no longer center aligned about the graphs. When I launch the shiny app locally with RStudio, everything is centered perfectly, but when going to the shinyapps.io address, it's off-center.

It looks like the text labels are shifted to the left. I've tried various changes with hjust, position_dodge, and grouping to no avail. Strangely, I've tried adding whitespace ('      ') before AND after the percent label text, and in both instances it shifts the label even further left, which is strange.

Should be noted again that when running the app locally through rstudio, everything is centered correctly. Also, if I use plotOutput/renderPlot instead of ggiraph, the text is centered correctly both locally AND on shinyapps.io. 

Hope there is a way for this to be fixed because I love ggiraph so far and prefer it to plotly!

I've reproduced the example here:  [shinyapps.io link](https://rg-strategies.shinyapps.io/test/)

Code to reproduce:
```
library(shiny)
library(shinydashboard)
library(dplyr)
library(ggiraph)
library(ggplot2)
library(tidyr)
library(ggthemes)
library(stringr)

test = structure(list(Q14 = c("Extremely Concerned", "Very Concerned", 
                              "Somewhat Concerned", "Just a Little Concerned", "Not Concerned at All", 
                              "Does Not Apply to Me", "Don’t Know"), 
                      n = c(401, 291, 212, 177, 91, 22, 6),
                      Percent = c(33, 24, 18, 15, 8, 2, 1), 
                      pctiTxt = c("33%", "24%", "18%", "15%", "8%", "2%", "1%")), 
                 row.names = c(NA, 7L), class = "data.frame")
rownmi = 'Q14'
# conventional rounding function where .5 always rounded up
round2 = function(x, n) {
  posneg = sign(x)
  z = abs(x)*10^n
  z = z + 0.5
  z = trunc(z)
  z = z/10^n
  z*posneg
}


dfplot = ggplot(test, aes(x = !!as.name(rownmi),y=Percent)) + 
  geom_col_interactive(
                       width = .5,
                       fill = '#1a476f',
                       aes(tooltip = paste0(n,' respondents'))) + 
  geom_text_interactive(aes(y = Percent, label = paste0(pctiTxt),group = !!as.name(rownmi)), size=10, 
                        vjust = -.5, hjust = .5,
            color = "#38434D", fontface = "bold") + 
  theme(rect = element_blank(),
        legend.position = 'none') +
  theme_calc()  +
  scale_fill_stata() +
  theme(rect = element_blank(),
        legend.position = 'none',
        text = element_text(colour = "black"),
        plot.title = element_text(size = 24), 
        legend.title = element_text(size = 10), 
        legend.text = element_text(size = 8), 
        axis.title = element_text(size = 10), 
        axis.text = element_text(size = 14,vjust = -.1)) +
  scale_y_continuous(breaks = c(0,20,40,60,80,100),
                     #labels = function(x) paste0(x,"%"),
                     expand = c(0,0),
                     limits = c(0,100)) + 
  scale_x_discrete(drop = FALSE, labels = function(x) str_wrap(x, width = 15)) +
  labs(title = paste0('Percentage Breakdown by ',rownmi)) + 
  ylab('') +
  xlab('')



ui = dashboardPage(
  dashboardHeader(title = 'hi'),
  dashboardSidebar(
    sidebarMenu(
      menuItem('Dashboard',tabName = 'dashboard', icon = )
      )
  ),
  dashboardBody(
    tags$head(
      tags$style(type="text/css", "text {font-family: sans-serif}")
    ),
    tabItems(
      tabItem(tabName = 'dashboard',
              fluidPage(
                fluidRow(
                  column(12,
                         'survey',
                         ggiraphOutput(outputId = 'ggComplete'))
                )
              ))
    )
  )
)


server <- function(input,output){
  output$ggComplete = renderggiraph({
    girafe(ggobj = dfplot,width_svg = 12, height_svg = 8)
  })
 # output$ggComplete = renderPlot({dfplot})
}

shinyApp(ui,server)

```
Session info:
```
> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] gdtools_0.2.2        stringr_1.4.0        ggthemes_4.2.0       tidyr_1.0.2          ggplot2_3.3.0       
[6] ggiraph_0.7.0        dplyr_0.8.5          shinydashboard_0.7.1 shiny_1.4.0.2       

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6      pillar_1.4.3      compiler_3.6.3    later_1.0.0       tools_3.6.3       uuid_0.1-4       
 [7] digest_0.6.25     jsonlite_1.6.1    lifecycle_0.2.0   tibble_3.0.0      gtable_0.3.0      pkgconfig_2.0.3  
[13] rlang_0.4.5       cli_2.0.2         rstudioapi_0.11   yaml_2.2.1        fastmap_1.0.1     withr_2.1.2      
[19] xml2_1.3.2        htmlwidgets_1.5.1 systemfonts_0.2.0 vctrs_0.2.4       grid_3.6.3        tidyselect_1.0.0 
[25] glue_1.4.0        R6_2.4.1          fansi_0.4.1       farver_2.0.3      purrr_0.3.3       magrittr_1.5     
[31] scales_1.1.0      promises_1.1.0    ellipsis_0.3.0    htmltools_0.4.0   rsconnect_0.8.16  assertthat_0.2.1 
[37] mime_0.9          xtable_1.8-4      colorspace_1.4-1  httpuv_1.5.2      stringi_1.4.6     munsell_0.5.0    
[43] crayon_1.3.4     
```

**Comment by davidgohel** [comment]
Hi,

Did you read that documentation about fonts:

https://davidgohel.github.io/ggiraph/articles/offcran/customizing.html#fonts

It's certainly what is happening.

From what I can see you are using font "Deja vu". I guess this is only available font on your platform (shinyapps.io) but I don't have this font so it does not display as expected. 

Could you try with Roboto for example?

**Comment by Dr-Fishopolis** [comment]
You are 100% right. It took me a while to figure it out, but once I embedded Roboto font everything was back to normal. 

I apologize for the needless issue! This is the first time I've actually deployed an application to the web instead of internally, so I'm blind to a lot of these css/html/javaScript issues that are probably quite trivial for an experienced developer. Regardless, appreciate the point in the right direction, and keep up the great work with your package.
